pep-8001.rst

```diff
@@ -97,7 +97,76 @@ The election will be supervised by Ernest W. Durbin III, The PSF Director of Inf
+    <p>
+    If you have any questions, you can post in <a
+    href="https://discuss.python.org/c/committers">the Committers
+    topic</a>, on <a href="mailto:python-committers@python.org">the
+    python-committers list</a>, or <a
+    href="mailto:ewdurbin@gmail.com">contact the vote administrator
+    directly</a>.
+    <p>
+    <h1>Options</h1>
+    <p>
+    We're selecting between seven PEPs, each proposing a different
+    governance model.
+    <p>
+    The options below include links to the text of each PEP, as well
+    as their complete change history. The text of these PEPs was
+    frozen on December 1, when the vote started. But if you looked at
+    the PEPs before that, they might have changed. For convenience,
+    the following is a complete list of changes since the official
+    review period started on November 16:
+    <p>
+    <ul>
+      <li><i>(no changes)</i></li>
+    </ul>
+    <p>
+    A "Further discussion" option is also included. It represents the
+    option of not making a choice at all at this time, and continuing
+    the discussion instead. Including this option lets us demonstrate
+    the core team's readiness to move forward.
+    <p>
+    If you think a proposal is a particularly bad idea, you can
+    express that by ranking it below "Further discussion". If you
+    think all of the proposals are better than further discussion,
+    then you should rank "Further discussion" last.
+
+Candidates (note: linebreaks are significant here)::
+
+    <a href="https://www.python.org/dev/peps/pep-8010/">PEP 8010: The Technical Leader Governance Model</a> (Warsaw) (<a href="https://github.com/python/peps/commits/master/pep-8010.rst">changelog</a>)
+    <a href="https://www.python.org/dev/peps/pep-8011/">PEP 8011: Python Governance Model Lead by Trio of Pythonistas</a> (Wijaya, Warsaw) (<a href="https://github.com/python/peps/commits/master/pep-8011.rst">changelog</a>)
+    <a href="https://www.python.org/dev/peps/pep-8012/">PEP 8012: The Community Governance Model</a> (Langa) (<a href="https://github.com/python/peps/commits/master/pep-8012.rst">changelog</a>)
+    <a href="https://www.python.org/dev/peps/pep-8013/">PEP 8013: The External Council Governance Model</a> (Dower) (<a href="https://github.com/python/peps/commits/master/pep-8013.rst">changelog</a>)
+    <a href="https://www.python.org/dev/peps/pep-8014/">PEP 8014: The Commons Governance Model</a> (Jansen) (<a href="https://github.com/python/peps/commits/master/pep-8014.rst">changelog</a>)
+    <a href="https://www.python.org/dev/peps/pep-8015/">PEP 8015: Organization of the Python community</a> (Stinner) (<a href="https://github.com/python/peps/commits/master/pep-8015.rst">changelog</a>)
+    <a href="https://www.python.org/dev/peps/pep-8016/">PEP 8016: The Steering Council Model</a> (Smith, Stufft) (<a href="https://github.com/python/peps/commits/master/pep-8016.rst">changelog</a>)
+    Further discussion
+
```

Comment:
Might it make sense to add a link to the GitHub repository containing the voter file and commit reference that should/will/was used for the voter roll? I recognize that additional users can be added once the poll commences. https://github.com/python/voters

Reply:
I've suggested an edit to link to it farther up.

Reply:
The link to the voter roll probably isn't necessary here, since the only people who will be able to see this text are people who have already successfully received an invitation to vote. But it's harmless in any case.